unified-installer/astra-unified-installer.sh

```diff
@@ -1851,11 +1851,20 @@ step_generate_astra_connector_yaml() {
 
     # Default memory limit
     memory_limit=2
-    snapshot_count=""
     if prompt_user_yes_no "Do you anticipate having more than 10,000 snapshots and backups existing at the same time at any point? "; then
+        snapshot_count="10000" # Default snapshot count
         prompt_user_number_greater_than_zero snapshot_count "Please estimate the maximum number of snapshots and backups you expect to have existing simultaneously within this cluster? (enter number value): "
         # Calculate estimated_memory and round up to the nearest integer
-        estimated_memory=$(echo "$snapshot_count 5000" | awk '{printf("%d\n", ($1/$2)+0.6)}')
```

Comment:
Double check my change here. I wasn't sure where the "0.6" was coming from

Reply:
Sounds like a way to force rounding up when the default behavior is to round to the closest integer (up or down)? Or something like that.

Reply:
Put this back to how it was after talking with Oscar, but removed the 0.6 thing. It was mostly ok as is.